plugins/LayerActions/main.lua

```diff
@@ -13,28 +13,40 @@ function clone()
 
   -- Make sure there is a next page, it has pdf background and no annotations or the annotations can be overwritten
   if currentPage == numPages then
-    app.msgbox("No next page. ", {[1] = "OK"})
+    app.openDialog("No next page. ", {"OK"}, "", true)
     return
   end
 
   local bgNextPage = docStructure["pages"][nextPage]["pageTypeFormat"]
   if bgNextPage ~= ":pdf" then
-    app.msgbox("Next page has no pdf background. ", {[1] = "OK"})
+    app.openDialog("Next page has no pdf background. ", {"OK"}, "", true)
     return
   end
 
-  local nextPdfPage = docStructure["pages"][nextPage]["pdfBackgroundPageNo"]
-  print(nextPdfPage)
-
   local isAnnotated = docStructure["pages"][nextPage]["isAnnotated"]
   if isAnnotated then
```

Comment:
This test does not work (it does not work on master either). @rolandlo do you know how to fix that?

Reply:
Yes, I see the problem. There is a typo in 
https://github.com/xournalpp/xournalpp/blob/a6ba9797f2d0a71508b4efcf820f24ac8681a9be/src/core/plugin/luapi_application.h#L1933-L1934
and 
https://github.com/xournalpp/xournalpp/blob/a6ba9797f2d0a71508b4efcf820f24ac8681a9be/src/core/plugin/luapi_application.h#L1977-L1978

"Annoated" should be "Annotated"

Reply:
It's a regression from https://github.com/xournalpp/xournalpp/commit/45062283113b7db70d5981d0223bf2d49c8beaac by the way.